playground/build.gradle

```diff
@@ -0,0 +1,7 @@
+apply plugin: 'kotlin'
```

Comment:
Consider configure `kotlin-multiplatform` plugin

Reply:
What for?